### ISSUE TITLE
Seekbar proccess change with webvtt

### ISSUE THREAD

**Issue by sangnguyen-it** [google/ExoPlayer#5029]
Preview seekbar, seek to posion. I want to get content when seek bar change in webvtt, show image
getTextComponent().addTextOutput(new TextOutput() {
            @Override
            public void onCues(List<Cue> cues) {}

But. not sync between seetbar and simpleExoplayer for show image 
Please help me.
Thanks you.

**Comment by ojw28** [comment]
It's a little hard to parse your question, but I think this is a duplicate of #3480. Marking as such. Please let me know if I misunderstood.

**Comment by nguyenvansang** [comment]
@ojw28 
thanks you.
You can help me. 
Have Exoplayer support get all list<Cue> content Webvtt play video?
Because it get one Cue when TextOutput render play video. 
I want to get all list content into SimpleExoPlayer. 

Img 15
00:02:25.000 --> 00:02:35.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=1400,0,100,56

Img 16
00:02:35.000 --> 00:02:45.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=1500,0,100,56

Img 17
00:02:45.000 --> 00:02:55.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=1600,0,100,56

Img 18
00:02:55.000 --> 00:03:05.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=1700,0,100,56

Img 19
00:03:05.000 --> 00:03:15.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=1800,0,100,56

Img 20
00:03:15.000 --> 00:03:25.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=0,56,100,56

Img 21
00:03:25.000 --> 00:03:35.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=100,56,100,56

Img 22
00:03:35.000 --> 00:03:45.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=200,56,100,56

Img 23
00:03:45.000 --> 00:03:55.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=300,56,100,56

Img 24
00:03:55.000 --> 00:04:05.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=400,56,100,56

Img 25
00:04:05.000 --> 00:04:15.000
trailer-secretsofthethreekingdoms-logo.jpg#xywh=500,56,100,56


Thanks you.
